content/en/certificate_authority/oidc-in-fulcio.md

```diff
@@ -19,6 +19,7 @@ Email-based OIDC providers use the user's email as the subject of the certificat
 
 * Dex (`oauth2.sigstore.dev/auth`)
     * GitHub (Note that this is the email of the user, not the GitHub username)
+    * GitLab
```

Comment:
Looks like Sigstore's Dex instance is not configured to work with GitLab:

![Screenshot 2023-08-28 at 3 52 27 PM](https://github.com/sigstore/docs/assets/8869764/b3f0198c-bb61-4d1c-bf95-7ec6ce753a7a)

Reply:
```suggestion
```